tests/features/step_definitions/browserstack-local-steps.js

```diff
@@ -0,0 +1,20 @@
+/*eslint-env node */
+
+var {defineSupportCode} = require('cucumber');

```

Comment:
__[jshint]__
* 'destructuring binding' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

<!-- Comment by [SideCI](https://sideci.com) -->